cmd/operator/main.go

```diff
@@ -145,6 +145,7 @@ func main() {
 	if err := validateFlags(); err != nil {
 		logger.Fatalf("invalid flags: %v", err)
 	}
+	logger.Debug(flagsToString())
```

Comment:
these also could be better as fields and msg="parsed configuration"

Reply:
That's true. The only thing about making them fields is that they will be alphabetically sorted instead of having related values together

Reply:
Yeah lets go with the current version for now